archivers/upx-devel/Portfile

```diff
@@ -0,0 +1,45 @@
+# $Id$
+
+PortSystem       1.0
+PortGroup        github 1.0
+
+github.setup     upx upx 5c0340f
+
+name             upx-devel
+version          20161019
+categories       archivers
+maintainers      openmaintainer
+license          GPL-2+
+
+description      compress or expand executable files
+long_description UPX is a free, portable, extendable, high-performance \
+                 executable packer for several executable formats
+homepage         https://upx.github.io/
+platforms        darwin
+checksums        ${distname}${extract.suffix} \
+                 rmd160  205df5485c93e452c591402cc579baad785d8ffd \
+                 sha256 28c859898e61025f65f4985162de2a1fdaa221fd46453e7b2ea4facd7a67060e 
```

Comment:
When there is only one distfile, there is no need to tell `checksums` what file the checksums are for; it knows. In this case, since you're fetching from git, there is no distfile, so the entire `checksums` stanza is unnecessary. If, per @neverpanic's comment below, you change to not fetching from git, then you would keep (and if necessary update) the checksums.